src/frontend/src/components/Button/Button.styles.ts

```diff
@@ -0,0 +1,30 @@
+import { tv } from "tailwind-variants";
+
+const buttonStyles = tv({
+    base: [
+        // Base styles
+        "inline-flex items-center justify-center whitespace-nowrap rounded-[var(--spacing-4)] gap-2.5 transition-colors",
+        "font-bold text-button leading-normal",
+        "cursor-pointer",
+        // Focus, disabled, and pressed states from react-aria-components plugin
+        "focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-muted-blue-400 focus-visible:ring-offset-2",
+        "disabled:opacity-50 disabled:cursor-not-allowed",
+        "pressed:scale-[0.98]",
+    ],
+    variants: {
+        variant: {
+            primary: "bg-teal-800 text-white hover:bg-teal-950",
+            outline: "border border-gray-300 bg-transparent hover:bg-gray-100",
+            orange: "bg-orange-200 text-orange-800 hover:bg-orange-300",
+        },
+        size: {
+            default: "h-9 px-5 py-2 min-w-9",
```

Comment:
Instead of default, let's make these entries "large", "medium" and "small", [using their respective figma sizing properties](https://www.figma.com/design/LAy4vzpOLf7YnXbB7pGfN8/BHA-ECHO?node-id=486-1070&m=dev). And then for the `defaultVariants` let's set size to "small" to accomodate mobile-first.

Reply:
Good observation, medium as the default seems like the way to go!